### PR TITLE
Add logo to readme

### PR DESCRIPTION
**Pull request by PietrH**
#270 created a logo for the package, this PR adds it to the readme. 

- [x] Check logo link
- [x] Add Nele as contributor
- [x] check failing test

### CODE REVIEW

**Comment by PietrH** [comment]
We could consider uploading a higher quality logo, but this is not default behaviour in usethis. Probably to keep the page loading time down.

**Comment by PietrH** [comment]
I'm not sure why tests are failing

File: README.Rmd [hunk 1]
```diff
@@ -13,7 +13,7 @@ knitr::opts_chunk$set(
 )
 ```
 
-# camtraptor
+# camtraptor <a href="https://inbo.github.io/camtraptor"><img src="man/figures/logo.png" align="right" height="137" alt="camtraptor website" /></a>
```
Comment (peterdesmet) [inline README.Rmd:16]: Is it a convention to wrap the logo in a link? Why not:

```suggestion
# camtraptor <img src="man/figures/logo.png" align="right" alt="logo" width="120">
```

**Review by peterdesmet — CHANGES_REQUESTED** [review]

File: README.Rmd (hunk 1 above)
Comment (PietrH) [reply README.Rmd:16]: I used `usethis::use_logo()`, No idea what the reasoning behind it is.

Comment (peterdesmet) [reply README.Rmd:16]: Ok, you can leave the link. It is used to link to the website from the README in the GitHub repo. The link is removed on the pkgdown website itself.

**Comment by PietrH** [comment]
Made a mockup of a version with a red background. 

camtraptor_logo_lexend_red

**Comment by PietrH** [comment]
Checked with https://www.color-blindness.com/coblis-color-blindness-simulator/

**Comment by PietrH** [comment]
There was a tiny issue on the right lower corner of the hexagon, where a part of the background was bleeding into the black border. I filled it in with black:

camtraptor_logo_lexend_red_fix

I've also included the sRGB colour profile, and stored it as an interlaced PNG for quicker loading. Not sure that those changes will matter at all because `usethis` converts the image with imagemagick anyway.

**Comment by NeleMullens** [comment]
Love the logo @PietrH, for me this is fine. Feels very Jurassic to me

**Comment by PietrH** [comment]
Ready for review, should be able to merge if we agree on the logo.

**Review by peterdesmet — APPROVED** [review]
Looks fantastic! 🦖 Thanks @NeleMullens!

**Review by damianooldoni — APPROVED** [review]

**Pull request merged**